model/task_lifecycle.go

```diff
@@ -775,48 +775,104 @@ func RestartItemsAfterVersion(cq *commitqueue.CommitQueue, project, version, cal
 	return catcher.Resolve()
 }
 
-// DequeueAndRestartForTask restarts all items after the given task's version, aborts/dequeues the current version,
-// and sends an updated status to GitHub.
+// DequeueAndRestartForTask restarts all items after the given task's version,
+// aborts/dequeues the current version, and sends an updated status to GitHub.
 func DequeueAndRestartForTask(cq *commitqueue.CommitQueue, t *task.Task, githubState message.GithubState, caller, reason string) error {
-	if cq == nil {
-		var err error
-		cq, err = commitqueue.FindOneId(t.Project)
+	mergeErrMsg := fmt.Sprintf("commit queue item '%s' is being dequeued: %s", t.Version, reason)
+	if t.Details.Type == evergreen.CommandTypeSetup {
+		// If the commit queue merge task failed on setup, there is likely a merge conflict.
+		mergeErrMsg = "Merge task failed on setup, which likely means a merge conflict was introduced. Please try merging with the base branch."
+	}
+
+	_, err := dequeueAndRestartItem(dequeueAndRestartOptions{
+		cq:           cq,
+		projectID:    t.Project,
+		versionID:    t.Version,
```

Comment:
suggestion to rename this to `itemVersionID` to make it clear that the item is represented by a version here

Reply:
Done.